sdk/spring/azure-identity-spring/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.11.0-beta.1 (Unreleased)
+## 1.11.0 (2021-11-24)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+Regular updates for dependency versions.
```

Comment:
1. What does `Regular updates for dependency versions.` mean? Does it mean `### Dependency Upgrades`? We didn't update spring-boot version. Maybe you mean Azure SDK version?
2. Is `This release is compatible with Spring Boot 2.5.0 - 2.5.4.` is need here?

Reply:
Yes, it means the Azure SDK version.
Sure, I will add the compatibility infomation.

Reply:
> Yes, it means the Azure SDK version.

How about:
1.  Move this into `### Dependency Upgrades` section? 
2. And list these versions? (Maybe not necessary for now.)

Reply:
I prefer not to, the bom will include the versions.